### DISCUSSION TITLE
Unable to upload ISO

### DISCUSSION THREAD

**Discussion by Demonium** [apache/cloudstack#10130] [Q&A] [answered]
##### ISSUE TYPE
 * Other

##### CLOUDSTACK VERSION
CloudStack 4.20.0

##### CONFIGURATION
Public Network
Gateway         | CIDR                | VLAN/VNI | Start IP          | End IP | 
192.168.1.254 | 192.168.1.0/24 | vlan://10   | 192.168.1.60 | 192.168.1.99

Management Network
Pod    | Gateway         | Netmask         | VLAN/VNI          | Start IP            | End IP 
Pod 1 | 192.168.1.254 | 255.255.255.0 | vlan://untagged | 192.168.1.100 | 192.168.1.200

Information about the System VM
Name | State | Agent state | System VM type | IP address | Private IP address | Link-local/Control IP address | Version | Host | Zone |  
  | s-5-VM | Running | Up | secondarystoragevm | 192.168.1.61 | 192.168.1.111 | 169.254.48.94 | 4.20.0.0 | server.local | Zone 1 |  
  | v-6-VM | Running | Up | consoleproxy | 192.168.1.62 | 192.168.1.108 | 169.254.10.211 | 4.20.0.0 | server.local | Zone 1

##### OS / ENVIRONMENT
I'm running on Rocky Linux 9.5
AMD Threadripper 7970x
256G RAM

##### SUMMARY
I have followed the tutorial below to install cloudstack 4.20 https://docs.cloudstack.apache.org/en/4.20.0.0/quickinstallationguide/qig.html#
Everything has been installed correctly the vm consoleproxy and secondarystorage are in green 
HTTPS has been setup
When I try to upload an ISO I have an warning :
Unable to reach the management server or a browser extension may be blocking the Network request.
Followed by an error
Failed to upload ISO. - Error: Network Error

In the console log of the browser I have the error below: 192.168.1.61/upload/903be432-485a-4e70-9ab8-04e9d925293b:1 net::ERR_CONNECTION_TIMED_OUT

my network is 192.168.1.0/24
the gateway is 192.168.1.254

All has been setup on a single machine
Nothing appear in the log I'm able to connect to the System VM with the IP Link-local provided from the main host

##### STEPS TO REPRODUCE
Follow the different steps in the guide

Name
State
Agent state
System VM type
IP address
Private IP address
Link-local/Control IP address
Version
Host
Zone

[s-5-VM](https://server.local.local:8443/client/#/systemvm/4d710733-a1c0-402f-b1d9-92b0fcf7c65d)	
Running
Up
secondarystoragevm	[192.168.1.61](https://server.local.local:8443/client/#/systemvm/4d710733-a1c0-402f-b1d9-92b0fcf7c65d)	192.168.1.111	169.254.48.94	4.20.0.0	[server.local](https://server.local.local:8443/client/#/host/a7e7d61a-ac54-487b-bdd0-43891e950adb)	[Zone 1](https://server.local.local:8443/client/#/zone/d8ba380b-f785-4097-8661-253ae7938fda)	

[v-6-VM](https://server.local.local:8443/client/#/systemvm/57faca37-cb57-44dd-ba17-c6f29d86867f)	
Running
Up
consoleproxy	[192.168.1.62](https://server.local.local:8443/client/#/systemvm/57faca37-cb57-44dd-ba17-c6f29d86867f)	192.168.1.108	169.254.10.211	4.20.0.0	[server.local](https://server.local.local:8443/client/#/host/a7e7d61a-ac54-487b-bdd0-43891e950adb)	[Zone 1](https://server.local.local:8443/client/#/zone/d8ba380b-f785-4097-8661-253ae7938fda)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by vladimirpetrov** [comment, marked as answer]
Hi @Demonium, it looks like your SSVM's IP address is not reachable (192.168.1.61) from the browser you are trying the upload. Please double check this, fix the connectivity and try again.
I am able to successfully upload ISO files in ACS 4.20 so I don't think there's a bug there.

**Comment by Demonium** [comment]
Hi,
I was able to ping the private ip but not the other ip from my laptop 
Next I was able to connect to the ssvm system and ping my router...
At this steps i'm thinking about a problem looking my network config... My local network is 192.168.1.0/24 so all the ip should be accessible, i'm I wrong ?

**Reply by DaanHoogland** [reply]
@Demonium are you still having this issue?

(I can only think of a conflict between your lan and your management network.)

**Reply by Demonium** [reply]
I didn't notice but thé upload of thé Iso is using https and thé certificate was not truste so I trust it and I was able to upload thé iso
